### PR TITLE
styling the label

### PR DESCRIPTION
**Pull request by marinanicolai**
@nicolad Is the added code permitted by the standards of tsx?

### CODE REVIEW

File: apps/expense/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": "@shared/tsconfig/nextjs.json",
-  "include": ["**/*.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["**/*.d.ts", "**/*.ts", "**/*.tsx", "components/InputArea/InputLabel.styled.js"],
```
Comment (nicolad) [inline apps/expense/tsconfig.json:3]: this is not needed

File: apps/expense/components/InputArea/InputLabel.styled.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+
+const InputLabel = styled.label`
+  font-size: 14px;
+  font-weight: bold;
+  color: #333;
+`;
+
+
+export default InputLabel;
```
Comment (nicolad) [inline apps/expense/components/InputArea/InputLabel.styled.js:12]: if u want to reuse this component across all the apps, u can move this to `ui` package and import from there

Comment (marinanicolai) [reply apps/expense/components/InputArea/InputLabel.styled.js:12]: @nicolad 
ok, thank you.

Can you please share your thoughts on the naming of the file?

Comment (nicolad) [inline apps/expense/components/InputArea/InputLabel.styled.js:12]: prettier is installed?

Comment (nicolad) [reply apps/expense/components/InputArea/InputLabel.styled.js:12]: too many spaces

Comment (nicolad) [reply apps/expense/components/InputArea/InputLabel.styled.js:12]: my preference: just `Label.tsx` and moved outside of `InputArea`

Comment (nicolad) [reply apps/expense/components/InputArea/InputLabel.styled.js:12]: don't mix `js` and `ts/ tsx`

**Review by nicolad — APPROVED** [review]

**Pull request merged**